Resources/Textures/SS220/Structures/barricades.rsi/meta.json

```
@@ -0,0 +1,47 @@
+{
+  "version": 1,
+  "license": "© SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt",
```

```suggestion
  "license": "EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt",
```
именно в таком виде, иначе не пройдёт тест